metron-platform/metron-writer/src/main/java/org/apache/metron/writer/hdfs/SourceHandler.java

```diff
@@ -128,28 +133,27 @@ public void run() {
     }
   }
 
+  // Closes the output file, but ensures any RotationActions are performed.
   protected void rotateOutputFile() throws IOException {
-    LOG.info("Rotating output file...");
+    LOG.debug("Rotating output file...");
     long start = System.currentTimeMillis();
     synchronized (this.writeLock) {
       closeOutputFile();
       // Want to use the callback to make sure we have an accurate count of open files.
       cleanupCallback();
-      this.rotation++;
 
-      Path newFile = createOutputFile();
-      LOG.info("Performing {} file rotation actions.", this.rotationActions.size());
+      LOG.debug("Performing {} file rotation actions.", this.rotationActions.size());
       for (RotationAction action : this.rotationActions) {
         action.execute(this.fs, this.currentFile);
       }
-      this.currentFile = newFile;
     }
     long time = System.currentTimeMillis() - start;
     LOG.info("File rotation took {} ms", time);
   }
 
   private Path createOutputFile() throws IOException {
-    Path path = new Path(this.fileNameFormat.getPath(), this.fileNameFormat.getName(this.rotation, System.currentTimeMillis()));
+    Path path = new Path(this.fileNameFormat.getPath(), this.fileNameFormat.getName(0, System.currentTimeMillis()));
```

Comment:
Can you explain this change a bit? Why are we going from `this.rotation` to `0`?

Reply:
Given that we don't rotate anymore, I just killed the variable and hardcoded it to 0.  There's never a rotation >0 that has data. I can change this (or comment) if it'd make it more clear.

Reply:
Might just be worth adding a doc / comment pass through here, since it's already being touched?

Reply:
Yes, you have explained it in the PR/JIRA, but it is a subtle point that would be good to call out in a comment.